README.md

```diff
@@ -7,25 +7,51 @@ Latest build: https://metamask.github.io/api-specs/latest/openrpc.json
 
 JSON-RPC API provided by MetaMask.
 
-MetaMask dapp developers lack a formal RPC specification. This attempts to document which methods MetaMask supports in a way that's versionable, human-readable and machine-readable. It will improve the accuracy of documentation, API, and clients.
+MetaMask dapp developers lack a formal RPC specification. This attempts to document which methods MetaMask supports in a way that's versionable, human- and machine-readable. It will improve the accuracy of documentation, API, and clients.
 
-MetaMasks effectiveness as an enterprise-grade application platform depends on its ability to provide a reliable and predictable developer experience.
+MetaMask's effectiveness as an enterprise-grade application platform depends on its ability to provide a reliable and predictable developer experience.
 
 ## Contributing
 
+> Before making any significant changes to the API spec, consider whether to first submit a [MIP](https://github.com/MetaMask/metamask-improvement-proposals/tree/main) and complete the MIP [workflow](https://github.com/MetaMask/metamask-improvement-proposals/blob/main/PROCESS-GUIDE.md).
+
 Edit the `openrpc.yaml` file.
 
-You can then run `yarn run build` to generate the output file again.
+Run `yarn build` to regenerate the output file: `dist/build/openrpc.json`.
 
-To test the result, you can paste the resulting file from `dist/build/openrpc.json` into [the OpenRPC playground](https://playground.open-rpc.org/) and view the result.
+> To view the result, paste that file's contents into [the OpenRPC playground](https://playground.open-rpc.org/).
 
 ### Build Process
-- filter out methods that are not supported/implemented by metamask
-- merge wallet methods with execution api methods
-- output to `build`
 
+To build, run:
+
+- `yarn install`, if you haven't setup the repo
+- `yarn build`
+
+The build process will:
+- filter out methods that are not supported/implemented by MetaMask
+- merge wallet methods with execution API methods
+- output to `dist` for release
+
+<details>
+  <summary>Build and merge summary</summary>
+    <div>
+	  The build includes the script `merge-openrpc.js`, this script:
+
+		- Loads openrpc.yaml and multichain/openrpc.yaml from the local file system.
+		- Fetches the Ethereum OpenRPC JSON spec from a [remote URL](https://raw.githubusercontent.com/ethereum/execution-apis/59e6a6f9947859e8bb41bc63b248aa026b0781bd/refs-openrpc.json) and filters the methods using filterExecutionAPIs.
+		- Merges the local MetaMask OpenRPC specification with the Ethereum OpenRPC specification.
+		- Adds tags to methods:
+			> Adds a tag to each method in the Ethereum OpenRPC specification.
+		- Writes out the merged and filtered OpenRPC specifications to temporary files:
+			- src/build/openrpc.json.
+			- src/build/multichain-openrpc.json.
+
+	These files are then output to the `dist` folder. 
+ </div>
+</details>
```

Comment:
These lists should probably be combined instead of repeated. Also, the dropdown list doesn't render properly:
<img width="1411" alt="Screenshot 2024-08-12 at 3 08 42 PM" src="https://github.com/user-attachments/assets/2d9f1422-09a7-4682-b46b-d3a8971302df">

Reply:
weird, vscode preview fine 
![image](https://github.com/user-attachments/assets/e632cde9-7a21-4bc7-ba29-6b36a1706114)

Reply:
GitHub also renders this page fine: 
<img width="411" alt="image" src="https://github.com/user-attachments/assets/b31ddc16-952e-4c36-aa9d-30efefa7533a">

Reply:
what are you previewing with?

Reply:
Oh, the dropdown is collapsed by default as expected. I'm noting that when you open the dropdown, most of the text appears in a code block, which I assume is not intended. This is probably due to indentation issues. I'm previewing on GH.

Reply:
ah -- it's an indenting issue --> Fixed